go.mod

```diff
@@ -4,11 +4,13 @@ go 1.17
 
 require (
 	github.com/evanphx/json-patch v4.12.0+incompatible
+	github.com/evanphx/json-patch/v5 v5.6.0
 	github.com/fsnotify/fsnotify v1.5.1
 	github.com/go-logr/logr v1.2.0
 	github.com/go-logr/zapr v1.2.0
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.17.0
+	github.com/pkg/errors v0.9.1
```

Comment:
Not sure how errors are usually wrapped in controller-runtime, but given that pkg/errors is only added with this PR it's probably not the same as in CAPI

Maybe we shouldn't introduce it (but let's see what CR maintainers think)